sample-app/src/components/Facet.tsx

```diff
@@ -15,11 +15,41 @@ interface FacetProps extends FacetTextConfig {
   searchable?: boolean,
   collapsible?: boolean,
   defaultExpanded?: boolean,
-  onToggle: onFacetChangeFn
+  onToggle: onFacetChangeFn,
+  customCssclasses?: FacetCssClasses,
+  cssCompositionMethod?: CompositionMethod
+}
+
+interface FacetCssClasses {
+  facetLabel?: string,
+  body?: string,
```

Comment:
what is `body` css class for?

Reply:
oops, I forgot to remove that after refactoring